package.json

```diff
@@ -44,6 +44,7 @@
     "release": "./script/upload.py -p",
     "repl": "python ./script/start.py --interactive",
     "start": "python ./script/start.py",
-    "test": "python ./script/test.py"
+    "test": "python ./script/test.py",
+    "test-asar-integrity": "./spec/node_modules/.bin/mocha ./spec/asar-integrity-test"
```

Comment:
As far I understand, normal tests run in a special prepared Electron instance and this instance is shared across all tests.

Integrity tests runs electron per each test assert with a special app, that exited on app ready event. So, not possible to run it as a normal tests.